css/style.css

```diff
@@ -44,20 +47,22 @@ a:visited {
 }
 
 a:hover {
+    background-color: rgba(255,0,0,1);
     box-shadow: 5px 10px rgba(0,0,0,1);
     font-weight: bold;
 }
 
 .footer {
-    margin: 30% auto;
+    position: absolute;
+    padding: 15px;
+    bottom: -8px;
 }
-/*-------------------------------------Table page --------------------------------------------*/
+
 .table-body {
     width: 100vw;
     height: 100vh;
```

Comment:
you're already applying this style to the `html` or the `body` tag, I don't think you need it here. Instead of `width: 100vw; height: 100vh;` you probably only need `height: 100%` - if `.table-body` is a `div` it will already be full width.